Fakebook.Profile/Fakebook.Profile.RestApi/Controllers/ProfileController.cs

```diff
@@ -75,6 +75,30 @@ public async Task<ActionResult<IEnumerable<ProfileApiModel>>> SelectProfilesAsyn
                 .ToList());
         }
 
+
+        /// <summary>
+        /// Action method that handles getting multiple profiles via their names;
+        /// PUT: /api/profiles/selection/{name}
```

Comment:
correct the comment,
and the producesresponsetype below

Reply:
Changed route to /search/{name} so that the route is not the same as the SelectProfilesAsync where we searched by email.  Deleted ProducesResponseTypes other than 200OK.